helm/odigos/templates/odigos-config-cm.yaml

```
@@ -11,6 +11,12 @@ metadata:
 data:
   config.yaml: |
     configVersion: 1
+    {{- if .Values.centralProxy.clusterName }}
+    clusterName: {{ .Values.centralProxy.clusterName }}
```

there is an error in the path here

```suggestion
    clusterName: {{ .Values.clusterName }}
```